### ISSUE TITLE
Models implementation

### ISSUE THREAD

**Issue by gxyau** [gxyau/cat_feat_enc_challenge#4]
Problems encountered so far for neural networks:
- [ ] nom_9 has 11k+ unique strings, too large for embeddings (Issue #3)
- [ ] decide on activation function
- [ ] metrics for loss function?
- [x] randomizer for neural networks (Issue #2)

Any more ideas for this?
